VirtoCommerce.Platform.Core/Notifications/SearchNotificationCriteria.cs

```diff
@@ -5,11 +5,7 @@ namespace VirtoCommerce.Platform.Core.Notifications
 {
 	public class SearchNotificationCriteria : ValueObject
     {
-        public SearchNotificationCriteria()
```

Comment:
To avoid breaking changes effect introduced by these changes I would like to recommend preserving previous default value for RepeatMinutesIntervalForFail property 5*60, and do not remove `RepeatHoursIntervalForFail ` and mark it as obsolete